tests/Makefile.am

```diff
@@ -216,6 +215,7 @@ if HAVE_UCS4
 dist_braille_specs_TESTS +=				\
 	braille-specs/akk.yaml				\
 	braille-specs/akk-borger.yaml			\
+	braille-specs/cuneiform-transliterated.yaml		\
 	braille-specs/hu-hu-g1-hyph_harness.yaml	\
 	braille-specs/uga.yaml
```

Comment:
See https://github.com/liblouis/liblouis/issues/1674. @egli Did you say you were going to look at it?

Reply:
I think there isn't enough time to fix #1674 in this release. That's why I just added `cuneiform-transliterated.yaml` to the UCS4 tests